src/rustllvm/RustWrapper.cpp

```diff
@@ -1226,6 +1226,10 @@ extern "C" LLVMValueRef LLVMRustBuildCall(LLVMBuilderRef B, LLVMValueRef Fn,
       unwrap(Fn), makeArrayRef(unwrap(Args), NumArgs), Bundles, Name));
 }
 
+extern "C" void LLVMRustSetCallNoTail(LLVMValueRef Call) {
+  dyn_cast<CallInst>(unwrap(Call))->setTailCallKind(CallInst::TCK_NoTail);
```

Comment:
Okay, then I'd make this `if (auto CI = dyn_cast<CallInst>(unwrap(Call))) CI->...` just so you can pass an invoke instruction.